lexicons/tools/ozone/moderation/defs.json

```diff
@@ -203,6 +203,36 @@
         }
       }
     },
+    "subjectView": {
+      "description": "Detailed view of a subject. For record subjects, the author's repo and profile will be returned.",
+      "type": "object",
+      "required": ["type", "subject"],
+      "properties": {
+        "type": {
+          "type": "ref",
+          "ref": "com.atproto.moderation.defs#subjectType"
+        },
+        "subject": {
+          "type": "string"
+        },
```

Comment:
I assume that `subject` is intended to contain the DID or AT-URI, is that right?

Would it be desirable to get tighter typings by doing what we do elsewhere, making this a union of a repo ref or record/strong ref?  Don't feel too strongly, this just looked like a new pattern to me and it does have the side effect of making the types a bit looser.

Reply:
> I assume that subject is intended to contain the DID or AT-URI, is that right?

yep. 

it is a different pattern than what we have been using but it's just soo much easier to pass around and handle compared to repo ref or strong ref.